acceptance/openstack/baremetal/v1/allocations_test.go

```diff
@@ -0,0 +1,45 @@
+// +build acceptance baremetal allocations
+
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	v1 "github.com/gophercloud/gophercloud/acceptance/openstack/baremetal/v1"
```

Comment:
This line can be removed - it's importing itself. I think this is what's causing the OpenLab failure.